README.md

```diff
@@ -86,6 +86,22 @@ WebAssembly backend.
 
 [wasm-pack]: https://rustwasm.github.io/wasm-pack/
 
+### Nix flake
+
+There is a Nix flake available in the `contrib` directory. With, you can get a
+reproducible build of Jotdown by cloning the repo and running
```

Comment:
"Reproducible build" usually means a bitwise identical binary, right? But is it necessarily reproducible in this regard? Even if all the inputs are identical, the output is not necessarily deterministic?

Reply:
"Reproducible" is typically the word that's used in the context of Nix, as it describes a build process that does not depend on (or get altered by) the build environment. 

I'm not sure whether the term "reproducible" entails "identical", but no; the binaries are not necessarily identical. However, that goes for different platforms as well. The point is that I can build a binary of Jotdown on my computer, and you can build a binary with the same behavior on another computer without any further setup.

Reply:
In our case I think we only know for certain that we have deterministic or reproducible _inputs_ and not necessarily a reproducible build or output.

E.g. https://reproducible.nixos.org/ seems to refer to "reproducible builds" as bitwise exact outputs.